### PR TITLE
Add docker resource limits to 20% services

### PR DESCRIPTION
**Pull request by YuryHrytsuk**
## Service updated
- Admin panels
- Adminer
- Depl. agent
- Filestash
- Graylog
- Jaeger
- Minio (:exclamation: Skipped since it is not used master / dalco / aws deployments)
- PG_BACKUP (:exclamation: https://github.com/kartoza/docker-pg-backup/issues/85)
- REDIS_COMMANDER

### CODE REVIEW

**Review by sanderegg — COMMENTED** [review]
thanks!

**Review by sanderegg — APPROVED** [review]

File: services/pg-backup/docker-compose.yml.j2 [hunk 1]
```diff
@@ -20,6 +20,13 @@ services:
       placement:
         constraints:
           - node.labels.pgbackup==true
+      resources:
+        limits:
+          memory: 6G
```
Comment (mrnicegyu11) [inline services/pg-backup/docker-compose.yml.j2:25]: This looks scarry, maybe we need to make an exception here. 6GB just for a database backup wow :D 🗡️

File: services/redis-commander/docker-compose.yml [hunk 2]
```diff
@@ -23,9 +23,11 @@ services:
         - traefik.http.routers.redis.middlewares=ops_auth@docker, ops_gzip@docker, ops_whitelist_ips@docker
       resources:
         limits:
-          memory: 128M
+          memory: 128M  # periodically hits this limit dalco-prod. Should we increase?
```
Comment (mrnicegyu11) [inline services/redis-commander/docker-compose.yml:26]: Let's increase to 192M or 256M then. Redis commander is actually used by the developers regularly. :--)

**Review by mrnicegyu11 — APPROVED** [review]
Thanks a lot, maybe have a look at my comments.

Very much needed contribution!

**Review by pcrespov — APPROVED** [review]

**Review by pcrespov — COMMENTED** [review]
thx! :tada:

**Comment by YuryHrytsuk** [comment]
It address the part of https://github.com/ITISFoundation/osparc-ops-environments/issues/330

File: services/admin-panels/docker-compose.yml.j2 [hunk 3]
```diff
@@ -97,7 +97,11 @@ services:
           - node.role == manager
       resources:
         limits:
-          memory: 2048M
+          memory: "2G"
+          cpus: "0.1"
```
Comment (GitHK) [inline services/admin-panels/docker-compose.yml.j2:101]: you will not be able to run anything in decent time with such low CPU limit

File: services/deployment-agent/docker-compose.yml.j2 [hunk 4]
```diff
@@ -22,6 +22,13 @@ services:
       replicas: 1
       placement:
         constraints: [node.role == manager]
+      resources:
+        limits:
+          memory: "400M"
+          cpus: "0.1"
```
Comment (GitHK) [inline services/deployment-agent/docker-compose.yml.j2:28]: very slow deploy agent as well

File: services/filestash/docker-compose.yml [hunk 5]
```diff
@@ -21,7 +21,11 @@ services:
         - traefik.http.routers.filestash.middlewares=ops_whitelist_ips@docker
       resources:
         limits:
-          memory: 512M
+          memory: 120M
+          cpus: "0.01"
```
Comment (GitHK) [inline services/filestash/docker-compose.yml:25]: while I can't comment on memory usage, the CPU for sure is very low. This one will go through a lot of files and will not respond at all in time.

File: services/graylog/docker-compose.yml [hunk 6]
```diff
@@ -11,6 +11,13 @@ services:
       replicas: 1
       restart_policy:
         condition: on-failure
+      resources:
+        limits:
+          memory: 1.2G
+          cpus: "0.1"
```
Comment (GitHK) [inline services/graylog/docker-compose.yml:17]: MongoDB with such low CPU, grayling will never reply in time. Also this one likes to have some ram so it replies faster to queries.

**Review by GitHK — COMMENTED** [review]
I think overall the settings look very low.
Q: did you try the limited services with datasets comparable to the ones were have on master?

File: services/redis-commander/docker-compose.yml (hunk 2 above)
Comment (YuryHrytsuk) [reply services/redis-commander/docker-compose.yml]: I have increased to 192M

File: services/pg-backup/docker-compose.yml.j2 (hunk 1 above)
Comment (YuryHrytsuk) [reply services/pg-backup/docker-compose.yml.j2]: Changed to 512M

File: services/graylog/docker-compose.yml (hunk 6 above)
Comment (YuryHrytsuk) [reply services/graylog/docker-compose.yml:17]: Updated to "1"

File: services/admin-panels/docker-compose.yml.j2 (hunk 3 above)
Comment (YuryHrytsuk) [reply services/admin-panels/docker-compose.yml.j2]: raised

File: services/adminer/docker-compose.yml [hunk 7]
```diff
@@ -25,8 +25,11 @@ services:
       resources:
         limits:
           memory: 128M
+          cpus: "0.01"
```
[the remark this answers is not in the document]
Comment (YuryHrytsuk) [reply services/adminer/docker-compose.yml]: raised

File: services/deployment-agent/docker-compose.yml.j2 (hunk 4 above)
Comment (YuryHrytsuk) [reply services/deployment-agent/docker-compose.yml.j2]: raised

File: services/filestash/docker-compose.yml (hunk 5 above)
Comment (YuryHrytsuk) [reply services/filestash/docker-compose.yml]: raised

**Review by GitHK — APPROVED** [review]

**Pull request merged**